lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -165,6 +186,21 @@ def pull_tests_from_queue(can_initialize_queue: false)
           test_file_paths
         end
 
+        def print_current_rspec_batch_command
+          test_file_paths = @queue.current_batch&.test_file_paths
+          return unless test_file_paths
+
+          puts
+          puts '=' * 80
+
+          order_option = @adapter_class.order_option(@cli_args)
+          printable_args = @rspec_pure.args_with_seed_option_added_when_viable(order_option, @rspec_runner.knapsack__seed, @cli_args)
+          messages = @rspec_pure.rspec_command(printable_args, test_file_paths, :batch_finished)
```

Comment:
It's not really true that the `:batch_finished`, but I guess it does not matter, right?

Reply:
Yes.

It would print  the `To retry the last batch of tests fetched from the Queue API, please run the following command on your machine:` message.